content/docs/2.9/scalers/aws-sqs.md

```diff
@@ -14,17 +14,20 @@ This specification describes the `aws-sqs-queue` trigger that scales based on an
 triggers:
 - type: aws-sqs-queue
   metadata:
-    # Required: queueURL
+    # Required: queueURL or queueURLFromEnv. If both provided, uses queueURL
     queueURL: https://sqs.eu-west-1.amazonaws.com/account_id/QueueName
+    queueURLFromEnv: QUEUE_URL # Optional. You can use this instead of `queueURL` parameter
     queueLength: "5"  # Default: "5"
     # Required: awsRegion
     awsRegion: "eu-west-1"
     identityOwner: pod | operator # Optional. Default: pod
+    
 ```
 
 **Parameter list:**
 
 - `queueURL` - Full URL for the SQS Queue. The simple name of the queue can be used in case there's no ambiguity.
+- `queueURLFromEnv` - the name of the environment variable on the scale target to read the queue URL from. You can use this instead of `queueURL` parameter
```

Comment:
```suggestion
- `queueURL` - Full URL for the SQS Queue. The simple name of the queue can be used in case there's no ambiguity. (Optional, You can use this instead of `queueURLFromEnv` parameter)
- `queueURLFromEnv` - Name of the environment variable on the scale target to read the queue URL from. (Optional, You can use this instead of `queueURL` parameter)
```

Reply:
No worries! done.